### ISSUE TITLE
Input field does not accept keyboard input all the time.

### ISSUE THREAD

**Issue by boykoc** [boykoc/meeting-timer#2]
Not sure exact issue at this time or how to consistently reproduce bug.

Scenario one:

- after page load, I can edit time
- start time
- stop time, try to edit, sometimes it works sometimes it doesn't

Scenario two:

- after page load, I can edit time
- start time
- stop time
- edit time
- start time
- stop time
- if can't edit, start time
- click in input field, can edit time

**Comment by boykoc** [comment]
Able to narrow down the issue. It occurs when user edits time then directly clicks start/stop button before clicking out of input field.

Steps to reproduce bug:

- load timer page
- click start
- click stop
- edit input
- **click start button**
- click stop
- try to edit input <= input wont accept key strokes

Current steps to work as expected/ by-pass bug:
- load timer page
- click start
- click stop
- edit input
- **click out of input but not a button**
- click start button
- click stop
- try to edit input <= input works as expected

**Comment by boykoc** [comment]
PR on the way.

**Comment by boykoc** [comment]
Narrowed down the issue. Under certain input combinations, when focus is given to the input it sets the cursor to the left of the input field even though there is a value already set in the field. This made it appear that the user input was not being accepted or registered. If a user happened to hit the `End` key or move the cursor with the right arrow key they would have been able to make edits.

It appears to happen under 2 scenarios, the one documented above and also if a user:

- loads page
- clicks start
- clicks in the input field to edit
- clicks start
- clicks stop
- tries to edit input <= cursor set to left on value

Basically, starting and stopping the timer inconsistently throws setting focus off. After trouble shooting the `timer.component.ts` with no luck (although there are some areas to cleanup later), I decided to handle this by setting the cursor in the `focus.directive.ts`.
